arelle/plugin/validate/NL/rules/nl_kvk.py

```diff
@@ -1120,6 +1123,47 @@ def rule_nl_kvk_4_1_5_2(
                   'Invalid filenames: %(invalidBasenames)s'))
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NL_INLINE_2024
+    ],
+)
+def rule_nl_kvk_4_4_3_1(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    NL-KVK.4.4.3.1: The extension taxonomy MUST not modify (prohibit and/or override) default members assigned to dimensions by the KVK taxonomy
+    """
+    for modelLink in cast(list[ModelLink], val.modelXbrl.baseSets[XbrlConst.dimensionDefault, None, None, None]):
+            if pluginData.isExtensionUri(modelLink.modelDocument.uri, val.modelXbrl):
```

Comment:
I think you've got an extra indent on this whole block, but doing things like this can prevent a ton of nested indentation:
```suggestion
            if not pluginData.isExtensionUri(modelLink.modelDocument.uri, val.modelXbrl):
                continue
            ...
```